packages/ui/src/lib/stores/persona.ts

```diff
@@ -4,7 +4,7 @@ import type { Post } from './post'
 import type { ReputationOptions } from '$lib/types'
 
 export interface Persona {
-	personaId: number | string
```

Comment:
I am not sure we need the `personaId`. The `personaId` is the key in the `personas.all` map so you can easily get it when iterating with `personas.all.forEach((persona, personaId) => {...`